tests/markdown_list/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2458,11 +2458,200 @@ proseWrap: "always"
 6. def
 7. ghi
 
+---
+
+0. abc
+0. def
+0. ghi
+
+---
+
+1. abc
+1. def
+1. ghi
+
+---
+
+2. abc
+2. def
+2. ghi
+
+---
+
+0. abc
+1. def
+2. ghi
+
+---
+
+0. abc
+1. def
+1. ghi
+
+---
+
+1. abc
+2. def
+3. ghi
+
+---
+
+2. abc
+3. def
+4. ghi
+
+---
+
+999. abc
+145. def
+69. ghi
+
+---
+
+0. abc
+
+---
+
+1. abc
+
+---
+
+2. abc
+
+---
+
+999. abc
+
+---
+
+0. abc
+1. def
+
+---
+
+1. abc
+2. def
+
+---
+
+
+1. abc
+1. def
+
+---
+
+2. abc
+3. def
+
+---
+
+999. abc
+1. def
+
+---
+
+999. abc
+2. def
+
 =====================================output=====================================
 5. abc
 6. def
 7. ghi
 
+---
+
+0. abc
+1. def
+2. ghi
+
+---
+
+1. abc
+1. def
+1. ghi
```

Comment:
Shouldn’t `0`-`1`-`1` be printed as `0`-`1`-`2` just like `1`-`2`-`2` is printed as `1`-`2`-`3`?

Reply:
No, we (not only me, it is common practice for many other developers) use `1` for git friendly markers (for example you want to start with `45` and have git friendly markers):
```md
45. list
1. list
1. list
1. list
1. list
```

Same for `0`, you may want start with `0`:
```md
0. list
1. list
1. list
1. list
1. list
```

Reply:
`1`-`2`-`2` is git friendly also but it is out of this PR because in out code we use only `1` as git friendly market, you can open an issue about this and i will fix it (it is easy)

Reply:
Why not do this then?

```markdown
0. list
0. list
0. list
0. list
0. list
```

Reply:
Because @ikatyang use `1.` for git friendly markers (and it is common practice) :smile: I think it is really other issue, developers use `1.` for git friendly markers usually (for example https://docs.microsoft.com/en-us/contribute/how-to-write-use-markdown, i can find more links if it is necessary)

Reply:
That makes sense. Thanks! 👍

Reply:
That makes sense, thanks! 👍